MapboxSearch/sample/src/main/java/com/mapbox/search/sample/AddressAutofillUiActivity.kt

```diff
@@ -160,13 +160,13 @@ class AddressAutofillUiActivity : AppCompatActivity() {
 
     private fun findAddress(point: Point) {
         lifecycleScope.launchWhenStarted {
-            val response = addressAutofill.suggestions(point, AddressAutofillOptions())
+            val response = addressAutofill.reverseGeocoding(point, AddressAutofillOptions())
```

Comment:
Look into how reverseGeocoding is used in address autofill. I'm not sure if it's meant to exist

Reply:
It was a mere rename, no functional change. I reverted it.